helpdesk-bot/src/main/resources/application.yaml

```diff
@@ -30,6 +30,7 @@ closeTicketSuccessResponse: "Thank you for contacting us, this session is now ov
 addMemberAgentSuccessResponse: "New member has been added to the help desk."
 addMemberClientSuccessResponse: "Your membership to the help desk has been updated by an agent."
 welcomeMessage: "Thanks for contacting the Equity Team at JPM. All messages sent in this chat will be delivered to the JPM Equity Team."
+noAgentsMessage: "There's no available agents to attend you ticket right now. Wait for a moment while we connect you to a new agent."
```

Comment:
"There are no available agents to attend your ticket right now. Please wait for a moment while we connect you to a new agent"

Reply:
Fix'd